epictrack-api/src/api/resources/work.py

```diff
@@ -128,7 +128,10 @@ class WorkResources(Resource):
     @profiletime
     def get():
         """Return all resource and work details"""
-        works = WorkService.find_allocated_resources()
+        is_active = request.args.get('is_active')
+        if is_active is not None:
+            is_active = is_active.lower() == 'true'
```

Comment:
do we need this if statement.

can we use this:
![image](https://github.com/bcgov/EPIC.track/assets/91914654/bf37cc6b-d23e-4836-884f-2628d31ef995)

or just args.get(is_active, False, bool)?